app.js

```diff
@@ -0,0 +1,184 @@
+const speakersData = [
+  {
+    name: 'David Stones',
+    image: './images/david.jpg',
+    title: 'Frontend Engineer @ Uber',
+    description:
+      'With years of experience, David will be enlightening us on the future of frontend development.',
+  },
+  {
+    name: 'Gift Akanji',
+    image: './images/gift.jpg',
+    title: 'Open source Developer @ FlightControl',
+    description:
+      'Haven contributed to over 100 open source projects, Gift will take us on how to become a successful open source contributor',
+  },
+  {
+    name: 'Michael Strings',
+    image: './images/michael.jpg',
+    title: 'Senior Full-Stack Developer @ Walmart',
+    description:
+      'Michael a succssful fullstack engineer would give us detailed information on becoming like him.',
+  },
+  {
+    name: 'Anita Burkes',
+    image: './images/anita.jpg',
+    title: 'Developer Advocate @ CloudFlare',
+    description: 'Anita would give us ensights into her field',
+  },
+  {
+    name: 'Jessica Thomas',
+    image: './images/jessica.jpg',
+    title: 'Head Developer Advocate @ Fusebit',
+    description:
+      'Jessica would speak to us about the upcoming release in JS and how it would make life easy for developers',
+  },
+  {
+    name: 'Lewis Philips',
+    image: './images/lewis.jpg',
+    title: 'Frontend Developer @ Platzi',
+    description:
+      'Lewis a successful frontend engineer would speak to us about his journey and JS as the language of the web.',
+  },
+  {
+    name: 'Blake Henderson',
+    image: './images/Blake.jpg',
+    title: 'Staff Developer Relations Engineer @ Firebase',
+    description:
+      'Blake would speak to us about the Development industry at large',
+  },
+  {
+    name: 'Sarah Nicoletti',
+    image: './images/nicoletti.jpg',
+    title: 'Director of Developer Advocacy @ Cloudinary',
+    description:
+      'Sarah would speak to us about the challenges facing the industry and how we can benefit from those challanges',
+  },
+];
+
+const menuOpenIcon = document.getElementById('menu_icon');
+const menuCloseIcon = document.querySelector('.close_menu_icon');
+const menuItems = document.querySelector('.menu_items');
+const li = document.querySelectorAll('.menu_items li');
+
+// functionality to open and close the menu bar
+
+menuOpenIcon.addEventListener('click', () => {
+  menuItems.style.display = 'flex';
+});
+
+menuCloseIcon.addEventListener('click', () => {
+  menuItems.style.display = 'none';
+});
+
+li.forEach((item) => {
+  item.addEventListener('click', () => {
+    menuItems.style.display = 'none';
+  });
+});
+
```

Comment:
- [ **OPTIONAL** ] What do you thing if you make the background of your mobile mobile menu to cover all the screen i.e. to have the `height: 100vh`. This wll make it looks nice.
![Screenshot from 2023-01-03 22-06-05](https://user-images.githubusercontent.com/83042742/210434254-aa48e7a7-52a1-44d9-b8c9-aa9db1a0ba99.png)